python/lsst/analysis/tools/actions/keyedData/keyedDataActions.py

```diff
@@ -128,7 +128,7 @@ def __call__(self, data: KeyedData, **kwargs) -> KeyedData:
             key_arg = key.format_map(kwargs)
             result[key_arg] = data[key_arg]
```

Comment:
```suggestion
            result[f"{self.identity}_{key_arg}"] = data[key_arg]
```

And you need to grep to see if this is used everywhere.